Os/test/ut/file/MyRules.cpp

```diff
@@ -0,0 +1,1150 @@
+// ======================================================================
+// \title Os/test/ut/file/MyRules.cpp
+// \brief rule implementations for common testing
+// ======================================================================
+
+#include "RulesHeaders.hpp"
+#include "STest/Pick/Pick.hpp"
+#ifdef __cplusplus
```

Comment:
See previous comment. ifdef cplusplus doesn't make sense if we are including hpp files. This is C++ code.

Reply:
Also I see C++ syntax in the file, so a C compiler won't compile it.